bootstrap.sh

```diff
@@ -6,14 +6,15 @@ if [[ `id -u` -eq 0 ]]; then
 fi
 
 
-declare -a RUBIES=('2.0' '2.1.10' '2.2.5' '2.3' 'jruby-1.7' 'jruby')
-DEFAULT_RUBY="2.0.0-p648"
+IOS_VERSIONS="8.1,8.2,8.3,8.4,9.0,9.1,9.2,9.3,10.0,10.1,10.2,10.3"
+declare -a RUBIES=('2.0' '2.1.10' '2.2.5' '2.3' '2.4' 'jruby-1.7' 'jruby')
```

Comment:
I know we've maintained a lot of these versions because of people specifically asking for them to be present and help reduce job start time, but I think it's ok to try trimming them for 8.3 and future releases, just make a _big_ note of it in the blog post